CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Features
 * **js:** Migrate tests from Karma to Jasmine browser runner (#899)
+* **assets:** Upgrade Inter font to version 4.1 (bedrock #15999)
```

Comment:
needs to be rebased and moved above release 19.3.0

Reply:
Q: Do the WOFF removals (now only WOFF2 left) need any migration notes, regarding the change in browser support? https://caniuse.com/woff2 vs. https://caniuse.com/woff

Reply:
(esp. since [#1030](https://github.com/mozilla/protocol/pull/1030/files) still seems to include WOFF 1.0 for future releases.)

Reply:
Good eye. I'm inclined to add migrate note and use woff2 only for new fonts, but have asked internally as well.

Reply:
We're going to go ahead with woff2 only. It's well supported and the fallback experience is acceptable for degraded support. I'll update the Mozilla typeface PR to remove woff.

@wen-2018 one last changelog request: we should call out that we are dropping support for `woff`